testsuite/tests/kuadrant/authorino/wristband/conftest.py

```diff
@@ -0,0 +1,117 @@
+    secret = openshift.create_signing_secret(wristband_secret_name, certificates["signing_ca"].key)
+    request.addfinalizer(lambda: openshift.delete_selector(secret))
+    return wristband_secret_name
+
+
+@pytest.fixture(scope="session")
+def certificates(cfssl, wildcard_domain):
+    """Certificate hierarchy used for the wristband tests"""
+    chain = {
+        "signing_ca": CertInfo(ca=True),
+    }
+    return cert_builder(cfssl, chain, wildcard_domain)
+
+
+@pytest.fixture(scope="module")
+def envoy(request, authorino, openshift, blame, backend, module_label, testconfig):
+    """Deploys Envoy with additional edge-route match"""
+    wristband_envoy = resources.files("testsuite.resources.wristband").joinpath("envoy.yaml")
+    envoy = Envoy(
+        openshift, authorino, blame("envoy"), module_label, backend, testconfig["envoy"]["image"], wristband_envoy
+    )
+    request.addfinalizer(envoy.delete)
+    envoy.commit()
+    return envoy
+
+
+@pytest.fixture(scope="module")
+def wristband_endpoint(openshift, authorino, authorization_name):
+    """Authorino oidc wristband endpoint"""
+    return f"http://{authorino.oidc_url}:8083/{openshift.project}/{authorization_name}/wristband"
+
+
+@pytest.fixture(scope="module")
+def authorization(authorization, wristband_secret, wristband_endpoint) -> Authorization:
+    """Add wristband response with the signing key to the AuthConfig"""
+    authorization.responses.add(
+        {
+            "name": "wristband",
+            "wrapper": "envoyDynamicMetadata",
+            "wristband": {
+                "issuer": wristband_endpoint,
+                "tokenDuration": 300,  # default value
+                "signingKeyRefs": [
+                    {
+                        "name": wristband_secret,
+                        "algorithm": "RS256",
+                    }
+                ],
+            },
+        }
+    )
+    return authorization
+
+
+@pytest.fixture(scope="module")
+def wristband_token(client, auth):
+    """Test token acquirement from oidc endpoint"""
+    response = client.get("/auth", auth=auth)
+    assert response.status_code == 200
+
+    assert response.headers.get("wristband-token") is not None
+    return response.headers["wristband-token"]
+
+
+@pytest.fixture(scope="module")
+def hostname_core(envoy, blame):
+    """Second route for the envoy"""
+    return envoy.add_hostname(blame("route-core"))
+
+
+@pytest.fixture(scope="module")
+def authorization2(openshift, blame, hostname_core, module_label, wristband_endpoint):
+    """Second AuthConfig with authorino oidc endpoint"""
+    authorization2 = AuthConfig.create_instance(
+        openshift, blame("auth-core"), None, hostnames=[hostname_core], labels={"testRun": module_label}
+    )
+    authorization2.identity.oidc("edge-authenticated", wristband_endpoint)
+    return authorization2
+
+
+@pytest.fixture(scope="module")
+def client2(hostname_core, envoy):
+    """Client for second AuthConfig"""
```

Comment:
I would again opt for better naming that reflects more its purpose, in both name and docstring